samples/snippets/get_table_labels_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+
+import get_table_labels
+
+if typing.TYPE_CHECKING:
+    import pytest
+
+
+def test_get_table_labels(
+    capsys: "pytest.CaptureFixture[str]",
+    table_id: str,
+) -> None:
+
```

Comment:
Let's set some labels like we do here: https://github.com/googleapis/python-bigquery/blob/main/samples/snippets/label_table.py#L29-L34

```suggestion
    table = client.get_table(table_id)
    table.labels = {"color": "green"}
    client.update_table(table, ["labels"])

```